android/assets/bdx/scenes/Scene.bdx

```diff
@@ -1 +1 @@
 "BOX"}, "mesh_name": null, "visible": true, "camera": {"type": "PERSP", "projection": [1.4375, 0, 0, 0, 0, 2.555555555555556, 0, 0, 0.0, 0.0, -1.002002002031864, -1, 0, 0, -0.20020020318640186, 0]}, "use_priority": false, "parent": null, "class": "Camera", "type": "CAMERA", "properties": {}}, "Sacky": {"transform": [1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.6182255744934082, 1.0], "use_priority": false, "parent": null, "instance": null, "physics": {"ghost": false, "group": 1, "margin": 0.03999999910593033, "mask": 255, "mass": 1.0, "compound": false, "restitution": 0.0, "body_type": "DYNAMIC", "friction": 0.5, "bounds_type": "BOX"}, "active": true, "class": "Sacky", "type": "MESH", "properties": {}, "mesh_name": "Sacky", "visible": false}, "Player": {"transform": [1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, -5.0, 2.5, 1.0], "use_priority": false, "parent": null, "instance": null, "physics": {"ghost": false, "group": 1, "margin": 0.03999999910593033, "mask": 65535, "mass": 1.0, "compound": false, "restitution": 0, "body_type": "NO_COLLISION", "friction": 0.5, "bounds_type": "BOX"}, "active": true, "class": "Player", "type": "EMPTY", "properties": {}, "mesh_name": null, "visible": true}, "Platform": {"transform": [1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0], "use_priority": false, "parent": null, "instance": null, "physics": {"ghost": false, "group": 1, "margin": 0.05999999865889549, "mask": 255, "mass": 0, "compound": false, "restitution": 0.0, "body_type": "STATIC", "friction": 0.5, "bounds_type": "TRIANGLE_MESH"}, "active": true, "class": "Platform", "type": "MESH", "properties": {}, "mesh_name": "Platform", "visible": true}, "G_Sacky": {"transform": [1.0, 0.0, 0.0, 0.0, 0.0, -6.397578431460715e-07, 1.0, 0.0, 0.0, -1.0, -6.397578431460715e-07, 0.0, 0.0, -3.1399710609392173e-10, 1.6163779497146606, 1.0], "use_priority": false, "parent": "Sacky", "instance": null, "physics": {"ghost": false, "group": 1, "margin": 0.03999999910593033, "mask": 255, "mass": 1.0, "compound": false, "restitution": 0.0, "body_type": "NO_COLLISION", "friction": 0.5, "bounds_type": "BOX"}, "active": true, "class": "G_Sacky", "type": "MESH", "properties": {}, "mesh_name": "G_Sacky", "visible": true}}, "materials": {"Sacky": {"opacity": 1.0, "backface_culling": true, "alpha_blend": "OPAQUE", "texture": null, "emit": 0.0, "shadeless": true, "color": [1.0, 1.0, 1.0]}, "Grass": {"opacity": 1.0, "backface_culling": true, "alpha_blend": "OPAQUE", "texture": "grass.png", "emit": 0.0, "shadeless": true, "color": [1.0, 1.0, 1.0]}, "MortBlock": {"opacity": 1.0, "backface_culling": true, "alpha_blend": "OPAQUE", "texture": "mort_block.png", "emit": 0.0, "shadeless": true, "color": [1.0, 1.0, 1.0]}, "G_Sacky": {"opacity": 1.0, "backface_culling": true, "alpha_blend": "OPAQUE", "texture": "sacky.png", "emit": 0.0, "shadeless": true, "color": [1.0, 1.0, 1.0]}}, "name": "Scene", "cameras": ["Camera"]}
```

Comment:
still have no idea if this file should be in version control or not. Smells like a build artifact, but I haven't verified.